src/meal.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react'
+
+const url = 'https://www.themealdb.com/api/json/v1/1/random.php'
+
+const Meal = () => {
+  const [food, setFood] = useState([])
+
+  const fetchFood = async () => {
+    const response = await fetch(url)
+    const data = await response.json()
+    setFood(data.meals)
+  }
+
+  useEffect(() => {
+    fetchFood()
+  }, [])
+
+  return (
+    <>
+      <div className="button">
+        <button onClick={() => fetchFood()} className="btn">
+          Generate Meal
+        </button>
+      </div>
+      <section className="meals">
+        {food.map((f) => {
+          // I wonder if there is an easier way to do this...or maybe not destructuring and then using dot notation to access the values...
```

Comment:
Instead of calling the map function within the return portion of the app, you can store it in the variable f above and then call it directly in the return portion